frontend/src/container/Account.jsx

```diff
@@ -105,10 +105,11 @@ export class Account extends React.Component {
     const { props } = this
     const fetchUserAgenda = await props.dispatch(getLoggedUserCalendar())
     switch (fetchUserAgenda.status) {
-      case 200:
+      case 200: {
```

Comment:
is this a new linting rule ?

Reply:
Yes : https://eslint.org/docs/rules/no-case-declarations

Reply:
ok, good. You can resolve this conversation